### PR TITLE
Update pt-BR.po

### PR DESCRIPTION
**Pull request by TheMemoir**
I reviewed and updated some text excerpts for Brazilian Portuguese.

## Description

I reviewed and updated some text excerpts for Brazilian Portuguese after I have read the full PO file for pt_br.

## Changes

Corrected and update terms which I observed were mistranslated.

## Additional information

Tested on: <!-- OS including version ->

### CODE REVIEW

File: static/lang/pt-BR.po [hunk 1]
```diff
@@ -1383,7 +1383,7 @@ msgstr "Imagem não encontrada: %s"
 #: source/common/modules/markdown-editor/renderers/readability.ts:39
 #, javascript-format
 msgid "Readability mode (%s)"
-msgstr ""
+msgstr "Modo de legibilidade"
```
Comment (nathanlesage) [inline static/lang/pt-BR.po:1386]: You are missing the `%s` placeholder here (for the actual algorithm used) — was this by accident or on purpose?

**Review by nathanlesage — COMMENTED** [review]
Hey, thank you very much for your help in translating! This looks already pretty good, but it seems as if you've accidentally forgotten a template string (see my comment below)? If you could have another look at that, this would be great! Then we can merge!

Awesome work :)

File: static/lang/pt-BR.po (hunk 1 above)
Comment (TheMemoir) [reply static/lang/pt-BR.po:1386]: Sorry. I forgot that. I'll correct it right away.

Comment (nathanlesage) [reply static/lang/pt-BR.po]: The `%s` is very simple: If the translatable string contains these two characters – here, it's `Readability mode (%s)`, this means that the `%s` will be replaced with some other string while the app runs.

In this case, `%s` will be replaced with the readability algorithm that is being used. I.e., in the app, `Readability mode (%s)` will become `Readability mode (dale-chall)`.

You can put this placeholder wherever it makes sense. You could, in theory, put it in front of your translation if that makes grammatical sense. I think here you can just modify the translation as such: `Modo de legibilidade (%s)` (but I don't speak Portuguese, so think whether `Modo de legibilidade (dale-chall)` makes grammatical sense first!)

Comment (TheMemoir) [reply static/lang/pt-BR.po]: I guess I got it right, now. Would please let me know? Tks.

**Comment by nathanlesage** [comment]
This looks great! Thank you very much! :)

Merging …!

**Pull request merged**